specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,93 @@
+# Usage Unit
+
+Usage Unit is the unit of measure for the usage or consumption of a resource or service. Usage Unit is used in conjunction with Quantity In Usage Unit to gain understanding of usage or consumption of a given product or service in an environment.
+
+The UsageUnit column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values when the ChargeType is 'Usage'. UsageUnit should be expressed as a single unit of measure.  Units of measure used in UsageUnit SHOULD adhere to the values and format requirements specified in the Recommended Values section below.
+
+The Usage Unit for a particular line item may differ from PricingUnit when providers use different units, unit increments, or columns to determine cost. The values in UsageUnit and QuantityInUsageUnit are often listed at a finer granularity or over a different time interval than the PricingUnit and QuantityInPricingUnit. While UsageUnit may appear to relate to pricing and cost, UsageUnit is focused on resource or service consumption.  It is essential not to confuse UsageUnit with PricingUnit which is the basis for determining cost. The UsageUnit column MUST not be used as the basis for determining values related to any pricing or cost metric.
```

Comment:
nit: In the past, we've tried to keep all the normative content in a single paragraph (or bulleted list) usually towards the end. This has normative content in the middle and also the last sentence of the final paragraph. Just calling out in case we think its worth making consistent. I don't feel strongly it MUST change but if we can change and keep the same messaging, we should.

Reply:
Agree with Udam here.  I would actually translate this into an example to show how they are different rather than declaring such ... ~

"""
Typically, PricingUnit and UsageUnit may appear synonymous but not always.  Below are a few examples showing ... 
"""